src/services/k8s/knative.go

```diff
@@ -80,15 +84,78 @@ func loadManifest(namespace string, commitSha string, project *project.Project,
 		UpdateTimestampAnnotationName: time.Now().Format(time.RFC3339),
 	}
 
+	envVarList, err := loadEnvFile(envFile)
+	if err != nil {
+		return nil, err
+	}
+
+	service.Spec.Template.Spec.Containers[0].Env = append(service.Spec.Template.Spec.Containers[0].Env, envVarList...)
+
 	return service, nil
 }
 
-func Apply(namespace string, commitSha string, config *rest.Config, project *project.Project, dockerImage docker.DockerImage) error {
+func loadEnvFile(envFile string) ([]corev1.EnvVar, error) {
+	var envVarList []corev1.EnvVar
+	if _, err := os.Stat(envFile); err != nil {
+		if (os.IsNotExist(err)) && (path.Base(envFile) == defaults.EnvVarFile) {
+			log.Info("No environment variables file .env.initium to Load!")
```

Comment:
Sorry @spchortis one last thing ;) 
```suggestion
			log.Infof("No environment variables file %s to Load!", defaults.EnvVarFile)
```